docs/en_US/Tutorial/SetupNniDeveloperEnvironment.md

```diff
@@ -68,7 +68,8 @@ It doesn't need to redeploy, but the nnictl may need to be restarted.
 #### TypeScript
 
 * If `src/nni_manager` will be changed, run `yarn watch` continually under this folder. It will rebuild code instantly.
-* If `src/webui` or `src/nasui` is changed, use **step 3** to rebuild code.
+* If `src/webui` will be changed, run `yarn start` under this folder, and change `MANAGER_IP` like ``//<Web UI IP>:8080/api/v1/nni`` in `src/start/const.ts`. The web UI will refresh automatically if code is changed.
+* If `src/nasui` is changed, use **step 3** to rebuild code.

```

Comment:
If `src/nasui` is changed, use `yarn start` to watch the change of the folder. Meanwhile, `server.js` needs to be launched to read files from `./mockdata`, which contains `graph.json` and other files needed.

Reply:
Update to let user run yarn start. But not understand what user needs to do to launch server.js.